src/UnitySampleProject/Assets/Scripts/Prefabs/Web3AuthWallet/TransferW3A.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using System.Numerics;
+using ChainSafe.GamingSDK.EVM.Web3AuthWallet;
+using ChainSafe.GamingWeb3;
+using ChainSafe.GamingWeb3.Build;
+using ChainSafe.GamingWeb3.Unity;
+using Nethereum.Hex.HexTypes;
+using Nethereum.RPC.Eth.DTOs;
+using Prefabs.Web3AuthWallet.UI;
+using Prefabs.Web3AuthWallet.Utils;
+using Scripts.Web3AuthWallet;
+using UnityEngine;
+using UnityEngine.UI;
+using Web3Unity.Scripts.Library.Ethers.Contracts;
+using Web3Unity.Scripts.Library.Ethers.JsonRpc;
+using Web3Unity.Scripts.Library.Ethers.Web3AuthWallet;
+
+public class TransferW3A : MonoBehaviour
+{
+    public Text responseText;
+    public string contractAddress;
+    public string toAccount = "0xdA064B1Cef52e19caFF22ae2Cc1A4e8873B8bAB0";
+    public string amount = "1000000000000000000";
+    public string contractAbi;
+    private Web3AuthWalletConfig _web3AuthWalletConfig;
+    private Web3AuthWallet _web3AuthWallet;
+    private Web3 _web3;
+    private EthereumService _ethereumService;
+
+
+    private async void Awake()
+    {
+        var projectConfig = ProjectConfigUtilities.Load();
+        _web3 = await new Web3Builder(projectConfig)
+            .Configure(services =>
+            {
+                services.UseUnityEnvironment();
+                services.UseJsonRpcProvider();
+                services.UseWeb3AuthWallet();
+            })
+            .BuildAsync();
+        _web3AuthWallet = new Web3AuthWallet(_web3.RpcProvider, _web3AuthWalletConfig);
+        _web3AuthWalletConfig = new Web3AuthWalletConfig();
+        _ethereumService = new EthereumService(W3AWalletUtils.PrivateKey, projectConfig.Rpc);
+    }
+
+    public void OnEnable()
+    {
+        // resets response text
+        responseText.text = string.Empty;
+    }
+
+    public async void Transfer()
+    {
+        // smart contract method to call
+        string method = "transfer";
+        W3AWalletUtils.OutgoingContract = contractAddress;
+
+        // connects to user's wallet to send a transaction
+        try
+        {
+            // connects to user's wallet to call a transaction
+            var contract = new Contract(contractAbi, contractAddress);
```

Comment:
I think with the latest changes this should be using the contract builder

Reply:
Thanks, Ryan. I'm assuming so as well. I'm just not sure how that is implemented now and couldn't find any documentation on this. If you have a link could you share it?